src/app/screens/PeerlyScreens/Components/RatingBar.tsx

```diff
@@ -28,33 +30,37 @@ const RatingBar: React.FC<RatingBarProps> = ({
           Rewards given by {rewardedByPeople} people
         </Text>
       </View>
-      <Pressable onPress={onPressObjection}>
-        <View style={styles.flagContainer}>
-          <View style={styles.flagIcon}>
-            <FlagIcon />
-          </View>
-        </View>
-      </Pressable>
+      {self ? null : (
+        <>
+          <Pressable onPress={onPressObjection}>
+            <View style={styles.flagContainer}>
+              <View style={styles.flagIcon}>
+                <FlagIcon />
+              </View>
+            </View>
+          </Pressable>
 
-      <View style={styles.sliderContainer}>
-        <Slider
-          disabled={disableSlider || isRewardAlreadyGiven}
-          style={styles.slider}
-          minimumValue={0}
-          maximumValue={5}
-          step={1}
-          value={reward}
-          onValueChange={value => setReward(value)}
-          thumbImage={require('../../../../assets/images/starIcon.png')}
-          minimumTrackTintColor="#FFD700"
-          maximumTrackTintColor="#DDD"
-        />
-        <View style={styles.labelsContainer}>
-          <Text style={styles.sliderLabel}>Good</Text>
-          <Text style={styles.sliderLabel}>Nice</Text>
-          <Text style={styles.sliderLabel}>Love</Text>
-        </View>
-      </View>
+          <View style={styles.sliderContainer}>
+            <Slider
+              disabled={disableSlider || isRewardAlreadyGiven}
+              style={styles.slider}
+              minimumValue={0}
+              maximumValue={5}
+              step={1}
+              value={reward}
+              onValueChange={value => setReward(value)}
+              thumbImage={require('../../../../assets/images/starIcon.png')}
```

Comment:
Can we please move this to icons file, like we did for other icons?

Reply:
Yes, I missed out it. I will add it and update.